Create ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md (#1965)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md

---
date: ""

> title: Hide dashboard availability select if user has no permission to request roles to populate it.

Reviewer: ```suggestion
title: Dashboard availability selection is hidden for users who do not have the required permission
```

> title: Disable dashboard availability select if user has no permission to request roles to populate it. (#6173) [GRAFT][release/y2024] (#6180)

Author: The Claude AI suggested to use the following title:
```suggestion
title: disable dashboard availability select for users without permission to request roles
```

> title: Disable dashboard availability select if user has no permission to request roles to populate it. (#6173) [GRAFT][release/y2024] (#6180)

Reviewer 2: ```suggestion
title: Hide dashboard availability select if user has no permission to request roles to populate it.
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57128
version: 1018.503.78
---

> Disable dashboard availability select if user has no permission to request roles to populate it. (#6173) [GRAFT][release/y2024] (#6180)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles necessary to populate it. With this change, the dashboard availability select is now disabled for users who do not have permission to request roles. This improves the user experience by avoiding confusion and preventing users from attempting to make changes they are not authorized to make.
```

> Disable dashboard availability select if user has no permission to request roles to populate it. (#6173) [GRAFT][release/y2024] (#6180)

Reviewer 2: ```suggestion
Previously, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles necessary to display availability select list. With this change, the dashboard availability select is now hidden for users who do not have permission to request roles. User need User management READ permission to see and modify availability property of dashboard.
```

> Previously, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles necessary to display availability select list. With this change, the dashboard availability select is now hidden for users who do not have permission to request roles. User need User management READ permission to see and modify availability property of dashboard.

Reviewer: ```suggestion
Previously, the dashboard availability selection was always visible for users, even if they did not have the permission to see it. With this change, the dashboard availability selection is hidden for users who do not have the required permission. Users must have User Management READ permission to see and modify the availability property of a dashboard.
```
